app/views/application/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="col-lg-12">
+  <div class="row">
+    <div class="col-md-4">
```

Comment:
Each of these divs and their contents are essentially the same code. Would it be possible to DRY this out using a very small partial template?